.vscode/settings.json

```diff
@@ -0,0 +1,35 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+  "prettier.bracketSpacing": true,
```

Comment:
According to [this comment](https://github.com/prettier/prettier/issues/528#issuecomment-276288879) this is the default setting so it shouldn't be necessary. I don't mind leaving it, just an FYI.

I'm generally not a huge fan of commiting editor configuration into a project in general, though. What's the use case for forcing this config on other developers?